Prox/Prox/PlaceDetails/PlaceDetailsIconInfoView.swift

```diff
@@ -32,15 +32,29 @@ class PlaceDetailsIconInfoView: UIView {
         return view
     }()
 
+    lazy var forwardArrowView = UIImageView(image: UIImage(named: "icon_forward"))
```

Comment:
This is a good idea. However, to save time I'll leave it as is and make it generic if we need to in the future.